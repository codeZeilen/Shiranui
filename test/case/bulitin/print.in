let unit = print(true);
let unit = print(false);
let unit = print(1);
let unit = print("Shiranui");
