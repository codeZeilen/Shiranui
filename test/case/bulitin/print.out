true
false
1
Shiranui
